src/main/java/ada/chore_api_v2/MissionChore/MissionChore.java

```diff
@@ -0,0 +1,53 @@
+package ada.chore_api_v2.MissionChore;
+
+import ada.chore_api_v2.Chore.Chore;
+import ada.chore_api_v2.Mission.Mission;
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "mission_chore")
+public class MissionChore {
+
+    @EmbeddedId
+    private MissionChoreId id;
+
+
+    @ManyToOne
+    @MapsId("missionId")
+    @JoinColumn(name = "mission_id", nullable = false)
+    private Mission mission;
+
+    @ManyToOne
+    @MapsId("choreId")
+    @JoinColumn(name = "chore_id", nullable = false)
+    private Chore chore;
+
+    @Column(name = "is_completed", nullable = false)
+    private Boolean isCompleted;
```

Comment:
```suggestion
    private Boolean isCompleted = false;
```
We can set this as false by default, since MissionChores will also start as not completed.